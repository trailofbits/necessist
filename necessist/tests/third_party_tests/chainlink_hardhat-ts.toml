url = "https://github.com/smartcontractkit/chainlink"
rev = "v1.13.0"
subdir = "contracts"
framework = "hardhat-ts"

[config]
ignored_functions = ["bigNumEquals", "evmRevert"]
