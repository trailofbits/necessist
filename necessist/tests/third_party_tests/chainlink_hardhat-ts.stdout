10485 candidates in 74 test files
